src/containers/permissions/PermissionsActionsGrid.js

```diff
@@ -20,28 +21,42 @@ const StyledActionsGrid :ComponentType<ActionsGridProps> = styled(ActionsGrid)`
   grid-template-columns: 2fr minmax(200px, 1fr) auto;
 `;
 
-const ObjectPermissionsActionsGrid = ({
+const PermissionsActionsGrid = ({
+  assignPermissionsText,
   onChangeFilterByPermissionTypes,
   onChangeFilterByQuery,
+  onClickAssignPermissions,
 } :{|
+  assignPermissionsText :string;
   onChangeFilterByPermissionTypes :(permissionTypes :PermissionType[]) => void;
   onChangeFilterByQuery :(query :string) => void;
+  onClickAssignPermissions :() => void;
```

Comment:
It's required because it's meant to be implemented in https://jira.openlattice.com/browse/APPS-2657 and https://jira.openlattice.com/browse/APPS-2658 and https://jira.openlattice.com/browse/APPS-2678.